sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -52,7 +52,10 @@ abstract class NativeDDLCommand(val sql: String) extends RunnableCommand {
  * unless 'ifNotExists' is true.
  * The syntax of using this command in SQL is:
  * {{{
- *    CREATE DATABASE|SCHEMA [IF NOT EXISTS] database_name
+ *   CREATE (DATABASE|SCHEMA) [IF NOT EXISTS] database_name
+ *     [COMMENT database_comment]
+ *     [LOCATION file_path]
```

Comment:
do we expect them to specify a full path (including extension) or a directory? Probably the latter?